NTUEEInfoDep/EESA-website
# EESA-#7 Timeline (show short intro for specific blog posts)

### What is this PR for?
1. Finish connect Contentful data to the chosen timeline
2. delete 2 other useless timeline

### What type of PR is it?
Improvement

### Todos
- [ ] css modify
- [ ] choose better icons
- [ ] a css related warning may need to be fixed

### What is the Github issue?
https://github.com/NTUEEInfoDep/EESA-website/issues/7

### How should this be tested?
1. In EESA main page, scrolling to the bottom, and you can see a timeline.
2. Add/Delete blog posts data in Contentful, and see the differences in timeline. 
    (**Currently, blog posts need to have a tag named '重要' to be showed on the timeline.**)

### Screenshots (if appropriate)
<img width="1158" alt="image" src="https://user-images.githubusercontent.com/73121038/168039653-8f5e457f-5ab9-4030-8d73-8d5f0d0af8dc.png">


### Questions:
* Do the license files need updating? No
* Are there breaking changes for older versions? No
* Does this need new documentation? No
